ajax/libs/selectonic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "selectonic",
+  "version": "0.6.3",
+  "description": "jQuery-plugin for making any list of items selectable by mouse and keyboard.",
+  "author": {
+    "name": "Alexey Novichkov",
+    "email": "mygrib@gmail.com"
+  },
+  "filename": "selectonic.min.js",
```

Comment:
Let's move this filed between `version` and `description` fields.
Because as usual, this filed help developers find the main file in first glance.
Thanks.